src/Stan/Inspection.hs

```diff
@@ -48,6 +55,20 @@ data Severity
 prettyShowInspection :: Inspection -> Text
 prettyShowInspection = show
 
+-- | Show 'Category' in a human-friendly format.
+prettyShowCategory :: Category -> Text
+prettyShowCategory cat = formatWith [magentaBg] $ "#" <> unCategory cat
+
+-- | Get the colour of the severity level.
+severityColour :: Severity -> Text
+severityColour = \case
+    Severe -> red
+    NotReallySevere -> yellow
```

Comment:
Looks like it's time to create an issue to think about possible severities and colours for them :slightly_smiling_face:

Reply:
Good idea: https://github.com/kowainik/stan/issues/30